themes/custom/droopler_theme/js/main-menu.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file
+ * Main menu operation.
+ */
+(function ($, Drupal) {
+
+  "use strict";
+
+  Drupal.behaviors.mainMenuMobileOperations = {
+    attach: function (context, settings) {
+      var $block = $('.we-mega-menu-submenu .block-block-content', context);
+      var $titles = $('.field--name-field-d-main-title', $block);
+
+      function showHideBlockContents() {
+        window.innerWidth < breakpointDesktop ? $contents.hide() : $contents.show();
+        $contents.removeClass(visibleClass);
+      }
+
+      if ($titles.length) {
+        var breakpointDesktop = 992;
+        var visibleClass = '.visible-content';
+        var blockContentClass = '.field--name-field-d-long-text';
+        var $contents = $titles.next(blockContentClass);
+
+        showHideBlockContents();
+        $(window).resize(showHideBlockContents);
```

Comment:
@grzegorz-pietrzak-droptica @js-droptica 
one thing came to my mind regarding this block :point_up: 
currently i scss we have breakpoint set to 1200 px,
plus this hiding of block content could be done in scss.. WDYT ?

Reply:
If it is possible, why not :)

Reply:
i think so..